ThunderBorg.py

```
@@ -245,62 +258,87 @@ def RawRead(self, command, length, retryCount = 3):
         fcntl.ioctl(self.i2cWrite, I2C_SLAVE, self.i2cAddress)
 
-
     def Print(self, message):
         """
-Print(message)
+        Print(message)
 
-Wrapper used by the ThunderBorg instance to print(messages, will call printFunction if set, print otherwise)
+        Wrapper used by the ThunderBorg instance to print(messages, will call printFunction if set, print otherwise)
         """
         if self.printFunction == None:
```

Ruff (0.14.7)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)